### DISCUSSION TITLE
Пора уже причесать мануал.

### DISCUSSION THREAD

**Discussion by Swinec** [bol-van/zapret#883] [Q&A]
не ютубом одним жив человек
можно поподробнее и с первых строк расписать как разделить стратегии для разных адресов?

что за переменная \<HOSTS\> где она вобще определяется?
man написан максимально не юзерфрендли. Сразу же мучаем стратегии, но нам же не надо крошить в фарш пакеты для ВСЕГО интернета.

**Comment by Fenrir-0xFF** [comment]
Дак вроде итак всё понятно написано: 
--new Означяет начяло новой стратегии, если стратегия самая первая то --new перед ней ненадо. 
Дальше пишиш конфиг для этой стратегии 
--hostlist=путь к списку с сайтами для которых будет применяться данная стратегия 
или
--hostlist-domains=список доменов через запятую к которым будет применяться данная стратегия
С фильтрами портов и протоколов думаю итак понятно
Пример для ютюба который работает у меня 
_**--new** --filter-tcp=443 --wssize=1:6 --dpi-desync=fake --dpi-desync-repeats=2 --dpi-desync-autottl=2 --dpi-desync-fake-tls=0x16030102 **--hostlist-domains=** youtube.com,googlevideo.com,ytimg.com,ggpht.com,youtubei.googleapis.com_
Сайты в нужной тебе стратегии не должны присутствовать в списках сайтов стратегий которые перед ней. 
Заметил что многие в конфигах --new пишут вконце строки, как по мне от этого только путаница у людей будет


@bol-van 
А что думаеш насчёт того чтобы сделать указание параметров для метода в скобках возле самого метода?  **метод(значение)** опции не целеком с их названием а только значения. 
TTL и desync-fooling можно так и оставить общими для всех, хотя на всякий случяй можно и тоже для конкретных методов указывать свои. 
Вроде проще, удобнее и нагляднее чем использовать отдельные опции 

Пример: --dpi-desync=syndata(файл или hex), wssize(значение), fake(протокол, файл или hex, Количество повторов), split2((значение split-pos),значение seqovl)
Там где множественные значения их в дополнительные скобки к примеру в split-pos
Если какоето значение нужно оставить поумолчянию и указать лишь значение следующего параметра то оставить просто запятую, чтобы смещения небыло.

**Comment by bol-van** [comment]
как сделано, так и сделано. загромождать чем-то еще не планирую

**Comment by bol-van** [comment]
переменной `<HOSTS>` нет. я не знаю откуда она взялась.
А плейсхолдеры `<HOSTLIST>` и `<HOSTLIST_NOAUTO>` описано в доке
